src/raven.js

```diff
@@ -469,6 +471,25 @@ Raven.prototype = {
     if (isErrorEvent(ex) && ex.error) {
       // If it is an ErrorEvent with `error` property, extract it to get actual Error
       ex = ex.error;
+    } else if (isDOMError(ex) || isDOMException(ex)) {
+      // If it is a DOMError or DOMException (which are legacy APIs, but still supported in some browsers)
+      // then we just extract the name and message, as they don't provide anything else
+      // We threat them the same, as they doesn't differ from each other enough to make them a separate checks
+      // https://developer.mozilla.org/en-US/docs/Web/API/DOMError
+      // https://developer.mozilla.org/en-US/docs/Web/API/DOMException
+      //
+      var name = ex.name || (isDOMError(ex) ? 'DOMError' : 'DOMException');
+      var message = ex.message ? name + ': ' + ex.message : name;
```

Comment:
capture exception has gotten quite long, maybe it would be good for this kind of logic to be in another method? 

so here we could do like

return `this._captureDomError(ex, options)` or something?

It's also fine how it is for the time being

Reply:
oh... that's good reason 😅